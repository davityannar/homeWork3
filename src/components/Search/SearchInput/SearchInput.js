import React from 'react';

import './SearchInput.scss';

const SearchInput = (props) => {
    


    return (
        <div>
            <input className='inputClass' placeholder = {props.placeholder} />
        </div>
    )
}

export default SearchInput;