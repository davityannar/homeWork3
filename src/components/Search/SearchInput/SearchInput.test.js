import React from 'react'
import SearchInput from './SearchInput'

describe('Tests SearchInput component', () => {
  it('Matching snapshot', () => {
    const component = shallow(<SearchInput />)

    expect(component).toMatchSnapshot();
  })
})