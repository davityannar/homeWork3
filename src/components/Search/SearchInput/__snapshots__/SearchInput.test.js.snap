// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests SearchInput component Matching snapshot 1`] = `
<div>
  <input
    className="inputClass"
  />
</div>
`;
